public/Filters.html

```diff
@@ -97,10 +164,10 @@
 
     </script>
     <form onsubmit="stop();" method="get">
-        <img class="logo" src="./logo.png" alt="UmtuWam">
-        <!-- <hr>
+        <!-- <img class="logo" src="./logo.png" alt="UmtuWam"> -->
```

Comment:
Just remove the commented-out code

Reply:
I see the changes here dont reflect what the actual changes are. Also the commented code here is left in because Jesse changes his mind on this every second day so i just switch between the 2 options.

The actual code on this line is 
`<!-- <img class="logo" src="./logo.png" alt="UmtuWam"> -->
        <hr>
        <h2>Preferences:</h2>
        <hr>`
        
I swap between the 2 whenever Jesse asks